### ISSUE TITLE
Is this alive?!

### ISSUE THREAD

**Issue by bardiarastin** [periodicaidan/dart_tdlib#3]
Hi, I want to make a Flutter Telegram client by using this, I wanted to know does this work correctly and fine? and is anyone interested to contribute?

**Comment by periodicaidan** [comment]
Yes it is very much alive, I just haven't committed anything to it in a while, as I've been working on other projects. I'm making this to make my own Telegram client in Flutter, in fact.

I can't guarantee it'll work in an actual project yet because for some reason TDLib needs to contact an IPv6 address, which I don't have enabled on my network (yet) so it just crashes after trying. Because of this, I can't test all the TDLib functions (though what I have tested works). I've also only tested it as a console application, not with Flutter.

Someone else has had problems with crashing (#2) but I haven't been able to recreate the crash myself so I don't know what's up with that.

You're very welcome to try out using it (instructions in the README). Once I get IPv6 working on my network I plan to work more on it.
